Automattic/wp-calypso
# Fix Sensei onboarding responsiveness

Related to p1683560600477689-slack-C02P7FHLVR9

## Proposed Changes

* It tweaks the image displayed as theme preview in order to not crop it and be with a fixed size.
* It tweaks the call-to-action position.
* It fixes some responsive styles.
* It also updates the site name placeholder to "Photography Class".

## Testing Instructions



* Navigate to http://calypso.localhost:3000/setup/sensei/senseiSetup
* Resize the window to different resolutions and make sure it works well.

## Pre-merge Checklist




## Screenshots

<img width="1516" alt="Screenshot 2023-05-12 at 18 32 32" src="https://github.com/Automattic/wp-calypso/assets/876340/111c9ea0-7500-4fb4-910a-47d5798b2979">
